src/Common/src/Microsoft/Win32/SafeHandles/SafeDirectoryHandle.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Win32.SafeHandles
```

Comment:
I don't recall how the conventions used elsewhere, but it's confusing to me that this is in Microsoft.Win32 but specific to Unix. Should the file name at least be SafeDirectoryHandle.Unix.cs? I'd prefer to keep SafeHandles that we only use on Unix in the Interop.<Lib> namespace.